src/v/config/configuration.cc

```diff
@@ -1714,6 +1714,13 @@ configuration::configuration()
       {.needs_restart = needs_restart::no, .visibility = visibility::user},
       {},
       &validate_non_empty_string_vec)
+  , kafka_topics_max(
+      *this,
+      "kafka_topics_max",
+      "Maximum number of Kafka topics per cluster. If `null`, then no limit is "
```

Comment:
```suggestion
      "Maximum number of Kafka user topics which can be created. If `null`, then no limit is "
```

since these are all cluster properties I don't think it makes much sense to say "per cluster".

Reply:
Switching to your suggestion